wiki/Game_modifier/Half_Time/en.md

```diff
@@ -0,0 +1,55 @@
+---
+stub: true
+tags:
+  - half time
+  - mod
+  - game modifier
+  - HT
+---
+
+# Half Time (mod)
+
+![HT mod icon](/wiki/shared/mods/HT.png "Half Time (HT) mod icon")
+
+*For the full list of all mods, see: [Game modifier](/wiki/Game_modifier)*
+
+## About
+
+- Abbreviation: HT
+- Type: Difficulty Reduction
+- Score Multiplier: • ![][o!s] ![][o!t] ![][o!c]: 0.30x • ![][o!m]: 0.50x
+- Default shortcut key: `E`
+- Caption: `Less zoom.`
+- Compatible game modes: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+
+## Description
+
+*Note: The method used to slow down the song may cause it sound muddy or robotic.*
+
+The **Half Time** mod is a [game modifier](/wiki/Game_modifier) that decreaes the overall [beatmap](/wiki/Beatmaps)'s speed (BPM) to 75% of the original, increases the length of the song by 33%, and decreases the [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty), [and health (HP)](/wiki/Beatmapping/Health) by a small amount.
+
+### osu!taiko
+
+In [osu!taiko](/wiki/Game_Modes/osu!taiko), the slowed song and BPM causes the notes to become very dense due to the decrease in AR. However, due to the way dendens are calculated in osu!taiko, they now take more hits to clear and cause them to award a **higher total value than without the mod** because denden hits are not affected by the Score Multiplier.
```

Comment:
```suggestion
In [osu!taiko](/wiki/Game_Modes/osu!taiko), the slowed song and BPM causes the notes to become very dense due to the decrease in AR. However, due to the way dendens are calculated in osu!taiko, they now take more hits to clear and cause them to award a **higher total value than without the mod** because denden hits are not affected by the score multiplier.
```

Reply:
maybe link to it too?

```suggestion
In [osu!taiko](/wiki/Game_Modes/osu!taiko), the slowed song and BPM causes the notes to become very dense due to the decrease in AR. However, due to the way dendens are calculated in osu!taiko, they now take more hits to clear and cause them to award a **higher total value than without the mod** because denden hits are not affected by the [score multiplier](/wiki/Game_modifier/Score_multiplier).
```